src/gitlab.js

```diff
@@ -32,68 +32,85 @@ export default class VerdaccioGitLab {
 
   }
 
+  static get logPrefix() { return '[gitlab]'; }
+
   authenticate(user: string, password: string, cb: Callback) {
+    this.logger.trace(VerdaccioGitLab.logPrefix, 'authenticate called for user:', user);
```

Comment:
I would prefer using `this.logger.trace('[gitlab]', ...` instead of `VerdaccioGitLab.logPrefix` as will not change at a later step.